hathor/indexes/manager.py

```diff
@@ -176,8 +282,8 @@ def enable_tokens_index(self) -> None:
 class RocksDBIndexesManager(IndexesManager):
     def __init__(self, db: 'rocksdb.DB') -> None:
         from hathor.indexes.memory_deps_index import MemoryDepsIndex
```

Comment:
Add a note that the RocksDB implementation is not ready to be used yet.

Reply:
There already is a comment bellow.

Reply:
Thanks!